spec/abstractops.html

```diff
@@ -818,29 +812,26 @@ <h1>ISO 8601 grammar</h1>
       UTCDesignator : one of
           `Z` `z`
 
-      Digits :
-          Digit Digits?
-
       DateFourDigitYear :
-          Digit Digit Digit Digit
+          DecimalDigit DecimalDigit DecimalDigit DecimalDigit
 
       DateExtendedYear :
-          Sign Digit Digit Digit Digit Digit Digit
+          Sign DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit
 
       DateYear :
           DateFourDigitYear
           DateExtendedYear
 
       DateMonth :
-          `0` NonzeroDigit
+          `0` NonZeroDigit
```

Comment:
Should that be NonZeroDecimalDigit upstream?

Reply:
Yes, I would consider that to be an improvement.